### ISSUE TITLE
Implement COE Punchlist

### ISSUE THREAD

**Issue by jason-gcio** [department-of-veterans-affairs/va.gov-team#37513]
## Background
COE is getting its last punchlist, before last check and final check

## Consideration
- Punch List from Jim: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/ebenefits/features/apply-for-home-loan-COE/research-design/coe-design-punchlist.md
- Form intro action link probably needs to be done first, as Kelli will ask (in another ticket)

## Tasks
- [x] Implement what can be done by the FE

## Acceptance Criteria
- [x] Punch list items have been addressed by FE
- [x] A PR has been requested for merging
